spec/unit/resource_helpers_spec.rb

```diff
@@ -0,0 +1,87 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers
```

Comment:
It must be working because the libraries are being mixed in on the top level and rspec is using the same object space across all the spec files, so because it's included in another spec file it usually works here. I say usually because if you only run this file with `rspec spec/unit/resource_helpers_spec.rb`, it will completely fail.